xdem/examples.py

```diff
@@ -94,9 +96,10 @@ def process_coregistered_examples(overwrite: bool =False):
     nuth_kaab.fit(reference_raster.data, to_be_aligned_raster.data,
                   inlier_mask=inlier_mask, transform=reference_raster.transform)
     aligned_raster = nuth_kaab.apply(to_be_aligned_raster.data, transform=reference_raster.transform)
```

Comment:
This line and the few lines below could be simplified now that Coreg methods take Rasters as arguments and with the arithmetic overloading functions !